src/plugins/presentation_util/tsconfig.json

```diff
@@ -7,9 +7,15 @@
     "declaration": true,
     "declarationMap": true
   },
-  "include": ["common/**/*", "public/**/*", "storybook/**/*", "../../../typings/**/*"],
+  "include": [
+    "common/**/*",
+    "public/**/*",
+    "storybook/**/*",
+    "../../../node_modules/@types/react-syntax-highlighter/**/*",
```

Comment:
Without this addition, the `scripts/type_check` check would give the following error:

```
ERROR src/plugins/presentation_util failed
      ../../../kibana/node_modules/@storybook/components/dist/syntaxhighlighter/syntaxhighlighter.d.ts:2:36 - error TS7016: Could not find a declaration file for module 'react-syntax-highlighter/dist/cjs/prism-light'. '/dev/shm/workspace/kibana/node_modules/react-syntax-highlighter/dist/cjs/prism-light.js' implicitly has an 'any' type.
        Try `npm i --save-dev @types/react-syntax-highlighter` if it exists or add a new declaration (.d.ts) file containing `declare module 'react-syntax-highlighter/dist/cjs/prism-light';`

      2 import ReactSyntaxHighlighter from 'react-syntax-highlighter/dist/cjs/prism-light';
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


      Found 1 error.
```

I'm not comfortable that this fix is the right one, but it does work.

Reply:
@elastic/kibana-core what do you think? ☝️